confidential/confidential.go

```diff
@@ -1,10 +1,18 @@
 package confidential
 
 import (
+	"bytes"
 	"crypto/sha256"
+	"encoding/binary"
+	"github.com/vulpemventures/go-elements/common"
+	"github.com/vulpemventures/go-elements/internal/bufferutil"
 	"github.com/vulpemventures/go-secp256k1-zkp"
 )
 
+const (
+	confidentialValue = 9
```

Comment:
We should export this value, I would rename it like:
```suggestion
	ElementsUnconfidentialValueLength = 9
```

Reply:
Done